### PR TITLE
Add Windows OS to Actions CI test

### LINKED ISSUE #377: CI Test addition of Windows environment is required.
We need to proceed with the test in the Windows environment.
There was an error due to the folder separator in the window, but it was not inspected.

### PR DESCRIPTION
**Pull request by TrustHenry**

### CODE REVIEW

**Review by kchulj — APPROVED** [review]

File: tests/Config.test.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ describe("Test of Config", () => {
         assert.strictEqual(config.database.database, "stoa");
         assert.strictEqual(config.database.port.toString(), "3306");
         assert.strictEqual(config.database.password.toString(), "12345678");
-        assert.strictEqual(config.logging.folder, "/stoa/logs");
+        assert.notStrictEqual(config.logging.folder, null);
```
Comment (Geod24) [inline tests/Config.test.ts]: Isn't there a path package that would abstract this away ?

Comment (TrustHenry) [reply tests/Config.test.ts]: I tried, but I couldn't find the package.

Comment (MichaelKim20) [reply tests/Config.test.ts]:
```suggestion
        assert.strictEqual(config.logging.folder, path.resolve(sdk.getInitCWD(), "/stoa/logs"));
```

Comment (TrustHenry) [reply tests/Config.test.ts]: @MichaelKim20 Thank you. I updated it.

**Review by MichaelKim20 — APPROVED** [review]

**Comment by TrustHenry** [comment]
Thank you!

**Pull request merged**